src/build_configurations.md

```diff
@@ -1,3 +1,7 @@
-# Overview
+# Build Configurations
 
+Our hardened OS is based on [Spectrum](https://spectrum-os.org/doc/) which is also an open-source project that aims to create a computer operating system, based on the principle of security by compartmentalization.
```

Comment:
Proposing "Our hardened OS is a build configuration based on ..."
I can add more info on what "a build configuration" is after we merge this contrib.

Reply:
Tried to rewrite it according to Brian's comment as well. You are very welcome to add more info :D